packages/systemd/var-run-tmpfiles.conf

```diff
@@ -2,3 +2,4 @@ d /run/cache 0755 root root -
 d /run/lock 0755 root root -
 L /var/lock - - - - /run/lock
 Z /var/lib/systemd 0755 root root -
+z /var/lib/systemd/random-seed 600 root root -
```

Comment:
We should also label this `secret_t` by adding an entry to `packages/selinux-policy/fs.cil`.

Reply:
Done